CHANGELOG.md

```diff
@@ -41,9 +41,11 @@ This change was introduced in [pull request #2385: Remove deprecated `govuk-main
 
 #### Remove deprecated `$govuk-border-width-form-element-error` variable
```

Comment:
I don't know if I'm getting confused, but I think there was some additional text here to explain why the change was made? I  think it was included in the first version of this PR, so potentially might have got lost when the newer changes were added? Or perhaps I'm imagining them completely 😅

Reply:
Haha, no, you're totally right - it's in the [release notes we tested with users](https://docs.google.com/document/d/1zXF8tRgbRJm8863-laNLB9ZcB4AbWdkNI-IGDtCth2U/edit?pli=1#heading=h.icv1a7bx25ly). Sorry, I might have lost it somewhere along the way. Will add back in.